src/digest.rs

```diff
@@ -47,14 +45,13 @@ pub fn hex_to_bytes(text: &str, dest: &mut [u8]) -> Result<()> {
 
 const N_BYTES: usize = 32;
 pub const DIGEST_NAME: &'static str = "SHA256SUM";
-pub use crypto::sha2::Sha256 as DigestComputer;
 
 pub fn create() -> DigestComputer {
-    sha2::Sha256::new()
+    Default::default()
 }
 
 
-#[derive(Clone,Copy,Debug,Eq,PartialEq)]
```

Comment:
Is there a reason for removing the Copy derivation here? I can't say that I have a great sense for the Rust customs about this sort of thing.